paywall/src/unlock.js/startup.ts

```diff
@@ -141,34 +141,41 @@ export default function startupWhenReady(
   window: Window,
   startupConstants: StartupConstants
 ) {
-  try {
-    // The presence of a cached account address indicates that the
-    // user has previously connected to MetaMask. In that case, it is
-    // acceptable for us to initialize the app right away. However, if
-    // there is no address in localStorage, we want to defer
-    // initializing the app until the user chooses to load the
-    // checkout modal.
-    const cachedAccountAddress = window.localStorage.getItem(
-      '__unlockProtocol.accountAddress'
-    )
-    if (!cachedAccountAddress) {
-      // We have to dispatch locked right away, otherwise it will
-      // never happen because we're stopping the rest of the app from
-      // loading.
-      ;(window as any).unlockProtocol = {
-        loadCheckoutModal: () => {
-          startup(
-            (window as unknown) as UnlockWindowNoProtocolYet,
-            startupConstants,
-            true
-          )
-        },
+  const web3Present = !!(window as any).web3
+
+  // Only try to do the deferred setup when we have a wallet. If we
+  // don't, we can do setup rright away, since it will just result in
```

Comment:
```suggestion
  // don't, we can do setup right away, since it will just result in
```

For once, I am the one telling you about a typo 🏅